app/policies/user_policy.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class UserPolicy < ApplicationPolicy
```

Comment:
Remove if we do not use it
Or move it to activeadmin spacename if we use it there